pkg/jobrunaggregator/jobrunaggregatorlib/ci_data_client.go

```diff
@@ -925,3 +927,33 @@ func GetUTCDay(in time.Time) time.Time {
 	year, month, day := in.UTC().Date()
 	return time.Date(year, month, day, 0, 0, 0, 0, time.UTC)
 }
+
+func (c *ciDataClient) ListAllKnownAlerts(ctx context.Context) ([]*jobrunaggregatorapi.KnownAlertRow, error) {
+	queryString := c.dataCoordinates.SubstituteDataSetLocation(
+		`SELECT *
+FROM DATA_SET_LOCATION.Alerts_AllKnown
```

Comment:
Who is updating this table? Is this a table that is automatically updated based on other table (e.g. Alerts) content?

Reply:
It is a view defined in BigQuery itself, here is the current query for it:

```sql
Select 
  AlertName, 
  AlertNamespace, 
  AlertLevel,
  Release, 
  min(JobRunStartTime) as FirstObserved, 
  max(JobRunStartTime) AS LastObserved, 
  count(*) AS Results
FROM `openshift-ci-data-analysis.ci_data.Alerts_UnifiedAlerts`
WHERE AlertSeconds > 0  
  AND (AlertNamespace LIKE 'openshift%' 
    OR AlertNamespace = "kube-system" 
    OR AlertNamespace = "default"
    OR AlertNamespace = "")
GROUP By AlertName, AlertLevel, AlertNamespace, Release
ORDER BY Release, AlertLevel, AlertName, AlertNamespace
```